ibc/src/component/connection.rs

```diff
@@ -279,12 +281,7 @@ pub trait View: StateExt + Send + Sync {
         connection: ConnectionEnd,
     ) -> Result<()> {
         self.put_domain(
-            format!(
-                "{}/connections/{}",
-                COMMITMENT_PREFIX,
-                connection_id.as_str()
-            )
-            .into(),
+            state_key::connection(COMMITMENT_PREFIX, connection_id),
```

Comment:
Since `COMMITMENT_PREFIX` is a crate-level constant, it would be cleaner if it were folded into the definition of `state_key::connection`, so that this call would be
```
state_key::connection(connection_id)
```
with the `COMMITMENT_PREFIX` inserted internally.